doc/classes/Node3D.xml

```diff
@@ -142,91 +145,96 @@
 			</description>
 		</method>
 		<method name="rotate_object_local">
 			<return type="void" />
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around axis, a unit [Vector3], by specified angle in radians. The rotation axis is in object-local coordinate system.
+				Rotates this node's [member basis] around the [param axis] by the given [param angle], in radians. This operation is calculated in local space (relative to this node) and preserves the [member position].
 			</description>
 		</method>
 		<method name="rotate_x">
 			<return type="void" />
 			<param index="0" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around the X axis by angle in radians.
+				Rotates this node's [member basis] around the X axis by the given [param angle], in radians. This operation is calculated in parent space (relative to the parent) and preserves the [member position].
 			</description>
 		</method>
 		<method name="rotate_y">
 			<return type="void" />
 			<param index="0" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around the Y axis by angle in radians.
+				Rotates this node's [member basis] around the Y axis by the given [param angle], in radians. This operation is calculated in parent space (relative to the parent) and preserves the [member position].
 			</description>
 		</method>
 		<method name="rotate_z">
 			<return type="void" />
 			<param index="0" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around the Z axis by angle in radians.
+				Rotates this node's [member basis] around the Z axis by the given [param angle], in radians. This operation is calculated in parent space (relative to the parent) and preserves the [member position].
 			</description>
 		</method>
 		<method name="scale_object_local">
 			<return type="void" />
 			<param index="0" name="scale" type="Vector3" />
 			<description>
-				Scales the local transformation by given 3D scale factors in object-local coordinate system.
+				Scales this node's [member basis] by the given [param scale] factor. This operation is calculated in local space (relative to this node) and preserves the [member position].
 			</description>
 		</method>
 		<method name="set_disable_scale">
 			<return type="void" />
 			<param index="0" name="disable" type="bool" />
 			<description>
-				Sets whether the node uses a scale of [code](1, 1, 1)[/code] or its local transformation scale. Changes to the local transformation scale are preserved.
+				If [code]true[/code], this node's [member global_transform] is automatically orthonormalized. This results in this node not appearing distorted, as if its global scale were set to the default. See also [method is_scale_disabled] and [method orthonormalize].
```

Comment:
"the default" isn't ideal here. A user could grab a scene with a node at a scale of (2, 3, 4), and then if instancing that and using editable children, the reset button in the inspector will set to a "default" of (2, 3, 4).

```suggestion
				If [code]true[/code], this node's [member global_transform] is automatically orthonormalized. This results in this node not appearing distorted, as if its global scale were set to [code](1, 1, 1)[/code]. See also [method is_scale_disabled] and [method orthonormalize].
```

Reply:
I'd rather reference the `Vector3.ONE` constant for a bit more maintainability. Maybe also saying "it's global scale were set to *the default of* [...]" but I'm not sure.